src/components/Drawer/Drawer.stories.tsx

```diff
@@ -0,0 +1,176 @@
+    },
+    footer: {
+      control: { disable: true },
+    },
+    adornment: {
+      control: { disable: true },
+    },
+  },
+} as Meta;
+
+const Footer = () => (
+  <Inline gap={SpaceSizes.md} justify="flex-end">
+    <Button size={SpaceSizes.lg} variant="outline">
+      Cancel
+    </Button>
+    <Button size={SpaceSizes.lg}>Confirm</Button>
+  </Inline>
+);
+
+const Adornment = () => (
+  <Icon color="neutral.900" name="long-arrow-left" type="ssc" />
+);
+
+export const Playground: Story<DrawerProps> = (args) => {
+  const [isShown, setIsShown] = useState(false);
+
+  return (
+    <Padbox paddingSize={SpaceSizes.xxl}>
+      <Inline justify="center">
+        <Button
+          size={ButtonEnums.ButtonSizes.lg}
+          onClick={() => setIsShown(!isShown)}
+        >
+          Open drawer
+        </Button>
+        {isShown && (
+          <Drawer
+            footer={<Footer />}
+            title="Title"
+            onClose={() => setIsShown(false)}
+            {...args}
+          >
+            <H2 style={{ margin: 0 }}>This is a placeholder</H2>
+            <Paragraph style={{ marginBottom: 0 }}>
+              You can replace this with a local component. You can also just
+              override the text, but keep in mind that you might lose content
+              that way in case we happend to change the underlying component.
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim veniam, quis nostrud exercitation ullamco laboris
+              nisi ut aliquip ex ea commodo consequat.
+            </Paragraph>
+          </Drawer>
+        )}
+      </Inline>
+    </Padbox>
+  );
+};
+
+export const WithTitleAndAdornment: Story = () => (
+  <Drawer
+    adornment={<Adornment />}
+    footer={<Footer />}
+    size="lg"
+    title="Large size drawer with title and adornment"
+    onClose={action('close-modal')}
+  >
+    <H2 style={{ margin: 0 }}>This is a placeholder</H2>
+    <Paragraph style={{ marginBottom: 0 }}>
+      You can replace this with a local component. You can also just override
+      the text, but keep in mind that you might lose content that way in case we
+      happend to change the underlying component. Lorem ipsum dolor sit amet,
+      consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+      dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+      ullamco laboris nisi ut aliquip ex ea commodo consequat.
+    </Paragraph>
+  </Drawer>
+);
+
+export const WithOverflowingTitle: Story = () => (
+  <Drawer
+    footer={<Footer />}
+    size="md"
+    title="This is a title that has no end, therefore it goes on and on until it fills all the space of its container and wants to go even further."
+    onClose={action('close-modal')}
+  >
+    <H2 style={{ margin: 0 }}>This is a placeholder</H2>
```

Comment:
We don't want to promote styling through `style` property much. You can wrap it in `Stack` and it should work the same way

Reply:
We probably can extract the Drawer content into a separate component like the `Footer`. So we don't need to repeat the same code in every story